content/chainguard/chainguard-images/working-with-images/scanners/grype-tutorial/index.md

```diff
@@ -0,0 +1,250 @@
+pip            23.0.1      23.3        python  GHSA-mq26-g339-26xf  Medium    
+python         3.10.14                 binary  CVE-2023-36632       High      
+python         3.10.14                 binary  CVE-2023-27043       Medium    
+python         3.10.14                 binary  CVE-2024-4030        Unknown   
+ssl_client     1.36.1-r28  1.36.1-r29  apk     CVE-2023-42365       Medium    
+ssl_client     1.36.1-r28  1.36.1-r29  apk     CVE-2023-42364       Medium
+```
+
+### Interpreting the Summary
+
+In the initial portion of its results output, grype summarizes information on the scanned artifact and gives an overview of known vulnerabilities. In the case of a scanned image, the output includes the image digest, a unique hash of the image that can be used as an identifier. 
+
+Overview output includes the number of packages, files, and executables found in the artifact. Generally speaking, CVE are detected against packages, but the number of executables detected can also give you an idea of the attack surface of the scanned image or filesystem.
+
+Finally, this portion gives a count of the number of CVE detected by severity and fixed status. Severity categorization sorts CVE into four categories based on the Common Vulnerability Scoring System (CVSS).
+
+{{< details "What is CVSS??" >}}
+{{< blurb/cvss >}}
+{{< /details >}}
+
+In our output, we can see that we have 0 critical, 1 high, and 8 medium CVEs:
+
+```
+   ├── by severity: 0 critical, 1 high, 8 medium, 0 low, 0 negligible 
+```
+grype also counts the number of CVE by fixed status. If a CVE is marked as fixed, it can be resolved by updating to a newer version of the package. Our output suggests that 7 packages have been fixed and can be remediated with updates:
+
+```
+   └── by status:   7 fixed, 3 not-fixed, 0 ignored 
+```
+### Itemized CVEs
+
+In addition to the summary, grype provides an itemized list of CVE. By default, these are in table format, and list the pakage name, current version, the package type (such as apt, apk, binary), and severity. If the package is fixed, grype will also indicate the package version where the fix was introduced.
+
+grype writes itemized CVE to stdout, so the report of itemized CVE can easily be redirected to a file:
+
+```bash
+grype python:3.10.14-alpine3.20 > report.txt
+```
+Alternately, you can use the `--file` flag to write to a file:
+
+
+```bash
+grype --file report.txt python:3.10.14-alpine3.20
+```
+
+Redirecting output can also be useful to suppress a long list of CVE so that the summary can more easily be viewed.
+
+## Output Formats
+
+### Standard Formats
+
+Itemized CVEs can be written to a number of formats, including the XML- or JSON-based [cyclonedx](https://cyclonedx.org/) SBOM standard and the [SARIF](https://sarifweb.azurewebsites.net/) static analysis format. To maximize the information provided by grype, it is recommended to use the JSON output type:
```

Comment:
```suggestion
You can use Grype to write itemized CVEs to a number of formats, including the XML- or JSON-based [cyclonedx](https://cyclonedx.org/) SBOM standard and the [SARIF](https://sarifweb.azurewebsites.net/) static analysis format. To maximize the information provided by grype, use the JSON output type:
```

Something you ought to work in is that I'm hearing a lot of passive voice in your writing. Remember: you're the expert! Feel free to just tell folks what they need to do, no need to prevaricate